websites/G/Google/presence.ts

```diff
@@ -86,7 +87,11 @@ presence.on("UpdateData", async () => {
 				// No default
 			}
 		}
+		if (privacy) {
+			delete presenceData.state;
+			if (presenceData.details.includes("Searching for"))
```

Comment:
Due to line 32, this will say "Viewing a Doodle:" with nothing below, so you should probably make another if statement to account for that

Reply:
I specifically just said on my comment above that my `if (privacy)` statement is contained and only affects while user is on `/search` lol. It does not effect the doodle case

Reply:
And `Viewing Doodle` is not privacy-invasive so I don't think you need one for that